crates/bevy_animation/src/lib.rs

```diff
@@ -63,17 +64,34 @@ pub struct EntityPath {
 #[derive(Reflect, FromReflect, Clone, TypeUuid, Debug, Default)]
 #[uuid = "d81b7179-0448-4eb0-89fe-c067222725bf"]
 pub struct AnimationClip {
-    curves: HashMap<EntityPath, Vec<VariableCurve>>,
+    curves: Vec<Vec<VariableCurve>>,
+    paths: HashMap<EntityPath, usize>,
     duration: f32,
 }
 
 impl AnimationClip {
     #[inline]
     /// Hashmap of the [`VariableCurve`]s per [`EntityPath`].
```

Comment:
```suggestion
    /// Vec<Vec<_>> of the [`VariableCurve`]s per [`EntityPath`].
```
Not sure that's the correct way of phrasing that

Reply:
maybe just remove the return type from the commnet.

Reply:
Done.